qemu/tests/cfg/netkvm_cpu_mapping.cfg

```diff
@@ -0,0 +1,12 @@
+- netkvm_cpu_mapping: 
+    virt_test_type = qemu
+    type = netkvm_cpu_mapping
+    only virtio_net
+    only Windows
+    vhost = on
+    image_snapshot = yes
+    no Win2k8..sp2
+    timeout = 360
+    driver_name = "netkvm"
+    cdroms += " virtio"
+    queues = ${smp}
```

Comment:
Note: If there are too many queues, more than 1024 files may be open. `vt_ulimit_nofile` may be necessary.

Acked-by: Yihuang Yu <yihyu@redhat.com>